content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,96 @@
+
+## Why we’re moving away from dockershim
+
+Docker was the first container runtime used by Kubernetes. This is one of the
+reasons why Docker is so familiar to many Kubernetes users and enthusiasts.
+Kubernetes uses dockershim as a container runtime interface (CRI) to let system
+components (like the kubelet) talk to Docker. It was built into Kubernetes code.
+Removing dockershim inevitably affects the entire CNCF family of projects beyond
+Kubernetes. Dependencies on Docker and dockershim have crept into various tools
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster and more
+secure than when Docker was the only choice. By removing the dockershim CRI,
+we're embracing the first value of CNCF: "[Fast is better than slow](https://github.com/cncf/foundation/blob/master/charter.md#:~:text=Fast%20is%20better%20than%20slow)".
+Migrating to another runtime will ensure that applications are more portable,
+with fewer dependencies on the infrastructure. Your apps will be more secure,
+with new features unlocked. A different runtime will also let us develop
+features like cgroupv2 support faster.
+
+## Deprecation timeline
+
+We announced dockershim deprecation in December 2020.  Full removal is targeted
+to Kubernetes 1.24, which is planned for April 2022. This deprecation timeline
+aligns with our [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior),
+which states that deprecated behaviors must function for no less than 1 year
+after their announced deprecation. We'll support version 1.23, which has
+dockershim, for another year in the Kubernetes project (and likely longer with
+managed Kubernetes providers). You'll have time to migrate. If you have more
+questions about the dockershim removal, refer to the [Dockershim Deprecation FAQ](https://k8s.io/dockershim).
+
+We asked you whether you feel prepared for the migration from dockershim in this
+survey: [Are you ready for Dockershim removal](https://kubernetes.io/blog/2021/11/12/are-you-ready-for-dockershim-removal/).
+We had about 600 responses. To everybody who took time filling out the survey,
+thank you.
+
+The results show that we still have a lot of ground to cover to help you to
+migrate smoothly. Several newer runtimes exist, and have been promoted
+extensively. However, a significant number of users still rely on dockershim,
+and sometimes have dependencies that need to be re-worked. Some of these
+dependencies are outside of your control. Based on the feedback received from
+you, here are some of the steps we are taking to help.
+
+## Our next steps
+
+Based on the feedback you provided, we'll be doing the following:
+
+- CNCF will sponsor work on updating documentation on how to migrate from
```

Comment:
[nit]

Do we want to link to the Github project tracking docs work, so that it's clear this work is already underway? Something like "To check the ongoing work, refer to [Docs - Dockershim Removal]\(https://github.com/orgs/kubernetes/projects/67)."?

Reply:
No, don't do that. :)